infra/helm-charts/mongo/charts/ingestbackup/README.md

```diff
@@ -20,16 +20,9 @@ Ingest infrastructure is deployed as a system of multiple self contained microse
 
 ### Quickstart
 
-Ingest Backup is deployed through Helm on Kubernetes through the `setup.sh` script provided in this module. The setup script [requires AWS access keys and role ARN](#credentials) can be downloaded through the AWS console in CSV format. With the addition of the [automated verification procedure](#verification), the setup script also requires the Slack Webhook URL where alerts can be sent. This can be retrieved through Slack apps management on their Web site.
```

Comment:
Is this not relevant anymore?

Reply:
No, this setup script is no longer needed and redundant with the mongo helm chart setup.sh. This is not the proper way to deploy the backup and verify cronjobs.